src/components/apartment/ApartmentRow.tsx

```diff
@@ -1,98 +1,115 @@
+import { Apartment, Project } from '../../types';
 import { fullURL } from '../../utils/fullURL';
 
 import styles from './ApartmentRow.module.scss';
 
-const BREAK_POINT = 768;
 const T_PATH = 'components.apartment.ApartmentRow';
 
 interface IProps {
   apartment: Apartment;
+  ownershipType: Project['ownership_type'];
 }
 
-const ApartmentRow = ({ apartment }: IProps): JSX.Element => {
-  const { apartment_number, apartment_structure, living_area, nid, url } = apartment;
-
+const ApartmentRow = ({ apartment, ownershipType }: IProps): JSX.Element => {
+  const { apartment_number, apartment_structure, living_area, reservations, url, apartment_uuid } = apartment;
+  const [rowOpen, setRowOpen] = useSessionStorage({ defaultValue: false, key: `apartmentRowOpen-${apartment_uuid}` });
   const { t } = useTranslation();
-  const [width, setWidth] = useState(window.innerWidth);
-  const [rowOpen, setRowOpen] = useSessionStorage({ defaultValue: false, key: `apartmentRowOpen-${nid}` });
-
-  const isDesktopSize = width > BREAK_POINT;
-  const isMobileSize = width <= BREAK_POINT;
-
-  const handleResize = () => setWidth(window.innerWidth);
 
-  const toggleRow = () => isMobileSize && setRowOpen(!rowOpen);
-
-  useEffect(() => {
-    window.addEventListener('resize', handleResize);
-    return () => window.removeEventListener('resize', handleResize);
-  }, []);
+  const toggleRow = () => setRowOpen(!rowOpen);
 
   const apartmentRowBaseDetails = (
     <>
       <strong>
         <span className="hiddenFromScreen">{t(`${T_PATH}.apartment`)}: </span>
-        <a href={fullURL(url)}>{apartment_number}</a>
+        {apartment_number}
       </strong>
       <span>
         <span className="hiddenFromScreen">{t(`${T_PATH}.ariaApartmentStructure`)}: </span>
         {apartment_structure}{' '}
-        {living_area && <span style={{ color: 'var(--color-black-60)' }}>({formattedLivingArea(living_area)})</span>}
+        {living_area && <span className={styles.apartmentLivingArea}>({formattedLivingArea(living_area)})</span>}
       </span>
-      {isMobileSize &&
-        (rowOpen ? (
-          <IconAngleUp style={{ marginLeft: 'auto' }} size={'m'} aria-hidden="true" />
-        ) : (
-          <IconAngleDown style={{ marginLeft: 'auto' }} size={'m'} aria-hidden="true" />
-        ))}
     </>
   );
 
-  const apartMentRowOtherDetails = (
-    <div>
-      <span className={isDesktopSize ? 'hiddenFromScreen' : styles.cellMobileTitle}>
-        {t(`${T_PATH}.applicants`)}&nbsp;{' '}
+  const renderReservationCountText = () => {
+    if (!reservations || reservations.length === 0) {
+      return <span className={styles.textMuted}>{t(`${T_PATH}.noReservations`)}</span>;
+    }
+    if (reservations.length === 1) {
+      return (
+        <span>
+          {reservations.length} {t(`${T_PATH}.reservation`)}
+        </span>
+      );
+    }
+    return (
+      <span>
+        {reservations.length} {t(`${T_PATH}.reservations`)}
```

Comment:
Could consider using i18next's [`pluralization support`](https://www.i18next.com/translation-function/plurals#singular-plural) for these.